pom.xml

```diff
@@ -518,6 +520,7 @@
             <id>build-sparkrapidsjni</id>
             <phase>validate</phase>
             <configuration>
+              <skip>${submodule.patch.skip}</skip>
```

Comment:
Why are we skipping building the C++ code if we're skipping patching?  This prevents building with subsequent changes on a followup build that refuses to proceed if it's already been patched.

Reply:
@gerashegalov this was your suggestion. I assume that it was to avoid building CUDF in an unpatched state.

Reply:
I tried to get rid of unnecessary work in the validate phase of the original PR assuming that we don't want to build anything unless the repo is patched.